Makefile

```diff
@@ -1,10 +1,25 @@
 PACKAGE_NAME = image-builder
 
+.PHONY: help
+help:
+	@echo "make [TARGETS...]"
+	@echo
+	@echo "This is the maintenance makefile of image-builder. The following"
+	@echo "targets are available:"
+	@echo
+	@echo "    help:               Print this usage information."
+	@echo "    build:              Build the project from source code"
+	@echo "    test:               Run tests"
```

Comment:
Hm this doesn't actually run the tests, it just compiles them.

Reply:
The target and corresponding help text was dropped, so this is now good to go.